### PR TITLE
Add missing translation strings (connect #2168)

### PR DESCRIPTION
**Pull request by muloem**
#### The solution

* We update with the latest translation strings for Flow UI

## Checklist
* [ ] Connect the issue
* [ ] Test plan
* [ ] Copyright header
* [ ] Code formatting
* [ ] Documentation

### CODE REVIEW

File: GAE/src/locale/es.properties [hunk 1]
```diff
@@ -96,7 +94,7 @@ Choose\ a\ pre-created\ cascade\ from\ the\ Cascade\ resources\ library.\ If\ yo
 Choose\ a\ question\ using\ the\ dropdown\ selectors\ above = Elija una pregunta utilizando el men� desplegable de arriba
 Choose\ an\ existing\ device\ group\ from\ the\ list = Seleccione un grupo de dispositivos existentes de la lista
 Choose\ cascade\ resource = Elija un recurso cascada
-Choose\ survey\ data\ to\ display = Seleccione datos de la encuesta para mostrar
+Choose\ data\ approval\ group = Eligir grupo de aprobaci�n
```
Comment (iperdomo) [inline GAE/src/locale/es.properties]: "Elija" instead "Elegir", similar to _Elija un recurso cascada_

File: GAE/src/locale/es.properties [hunk 2]
```diff
@@ -259,13 +257,12 @@ List\ of\ permissions = Lista de permisos
...
 Log\ in = Iniciar sesi�n
 Log\ out = Salir
-Login = Iniciar sesi�n
 Longitude = Longitud
 Make\ monitor\ group = Hacer grupo de monitoreo
-Manage = Administrar
 Manage\ API\ keys = Administrar c�digos de acceso API
 Manage\ Cascade\ Resources = Administrar recursos cascada
 Manage\ Devices\ and\ Assignments = Administrar dispositivos y asignaciones
+Manage\ data\ approvals = Gestionar la aprobaci�n de datos
```
Comment (iperdomo) [inline GAE/src/locale/es.properties]: "Administrar" is more LATAM oriented than "Gestionar"

File: GAE/src/locale/es.properties (hunk 1 above)
Comment (valllllll2000) [inline GAE/src/locale/es.properties]: @Ivan is it "Elija grupo de aprobación" or  "Elija el grupo de aprobación"?

Comment (ivan) [inline GAE/src/locale/es.properties]: Please don't @ivan, I get an email, wrong ivan

Comment (valllllll2000) [inline GAE/src/locale/es.properties]: oups sorry!

Comment (iperdomo) [inline GAE/src/locale/es.properties]: "Elija un" or "Elija el" should be fine. "Elija grupo de aprobación" feels _too_ imperative

**Pull request merged**